src/content-tags-drawer/ContentTagsDrawer.scss

```diff
@@ -0,0 +1,9 @@
+.pgn__sheet-component:has(#content-tags-drawer) {
+  min-width: max(500px, 33vw);
+}
+
+@media only screen and (max-width: 500px) {
+  .pgn__sheet-component:has(#content-tags-drawer) {
+    min-width: 100vw;
+  }
+}
```

Comment:
Could you raise a corresponding issue in paragon? It does seems like a severe limitation, and perhaps there can be support for different sizes for the sidebar as well.

Reply:
Done: https://github.com/openedx/paragon/issues/3023